tests/t314-basis.c

```diff
@@ -89,8 +90,15 @@ int main(int argc, char **argv) {
       CeedVectorRestoreArrayRead(u, &u_array);
       CeedVectorRestoreArrayRead(u_q, &u_q_array);
     }
-    CeedScalar tol = GetTolerance(CEED_SCALAR_TYPE, dim);
-    if (fabs(sum_1 - sum_2) > tol) printf("[%" CeedInt_FMT "] %0.12f != %0.12f\n", dim, sum_1, sum_2);
+    {
+      CeedScalarType scalar_type;
+
+      CeedGetScalarType(&scalar_type);
```

Comment:
I honestly don't know why this function was added, just that its there and untested on main :shrug:

Reply:
`CeedGetScalarType` was added for the Julia bindings: see discussion in https://github.com/CEED/libCEED/pull/788 as to why it needed to be function available at runtime.

As for why it doesn't take a `Ceed` object, that's because the scalar type can only be set for the entire library at (library) compile time, so all `Ceed` objects will have the same scalar type.

Reply:
Gotcha. Then is it also tested in the Julia library?

Reply:
It's just used to configure your Julia package based on the scalar type used to compile your libCEED library. I know Will at least did some manual testing that reconfiguring to single precision and running the Julia tests worked, but I don't think that's part of the CI anywhere?

Reply:
I think its tested there, but not in a case covered by LCOV